### PR TITLE
List continued

### PR DESCRIPTION
**Pull request by jacobrienstra**
Icons, title, subtitles, lines & height

*Need to update the readme but wanted to get the code pushed

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -27,6 +24,7 @@
 			}
 		],
 		"string-quotes": "single",
-		"length-zero-no-unit": null
+		"length-zero-no-unit": null,
+		"no-descending-specificity": true
```
Comment (rijkvanzanten) [inline .stylelintrc.json]: I'd swear this was already on through one of the extends 🤔 
Oh well, better safe than sorry 🤷‍♂

File: src/components/v-button/readme.md [hunk 2]
```diff
@@ -46,9 +46,10 @@ The button has a loading state that can be enabled with the `loading` prop. By d
 The loading slot is rendered _on top_ of the content that was there before. Make sure that your loading content doesn't exceed the size of the default state content. This restriction is put in place to prevent jumps when going from and to the loading state.
 
 ## Props
+
 | Prop       | Description                                                               | Default  |
 |------------|---------------------------------------------------------------------------|----------|
-| `block`    | Enable ull width (display block)                                          | `false`  |
+| `block`    | Enable null width (display block)                                         | `false`  |
```
Comment (rijkvanzanten) [inline src/components/v-button/readme.md]: `null` width? More like, `full` width 😉 

```suggestion
| `block`    | Enable full width (display block)                                         | `false`  |
```

File: src/styles/_colors.scss [hunk 3]
```diff
@@ -210,6 +210,7 @@ body {
 	--grey-900: #212121;
 
 	--blue-grey: #607d8b;
+	--blue-grey-25: #F5F7F8;
```
Comment (rijkvanzanten) [inline src/styles/_colors.scss]: @benhaynes thoughts on this?

On the one side, all numbered colors are exactly material design palette, so I'm a little hesitant to start adding more colors in there, as that would "officially" stop making this the material palette, but on the other hand, we already had to add "off white" so you could argue this is a cleaner solution for the same problem

Comment (rijkvanzanten) [reply src/styles/_colors.scss]: Discussed this on Slack, it's the way to go. I'm going to replace off-white in favor of this

File: src/components/v-list/v-list-item.vue [hunk 4]
```diff
@@ -91,24 +109,44 @@ export default defineComponent({
...
+			}
 		}
-
-		.v-list.dense &,
-		&.dense {
-			&:not(:last-child):not(:only-child) {
-				--v-list-item-margin-bottom: 4px;
+		.v-list.nav {
+			--v-list-item-padding: 0 8px;
+			--v-list-item-border-radius: 4px;
```
Comment (rijkvanzanten) [inline src/components/v-list/v-list-item.vue]: This border radius isn't set, maybe this `@at-root` business doesn't work?

File: .stylelintrc.json (hunk 1 above)
Comment (jacobrienstra) [reply .stylelintrc.json]: it was. i turned it off then on again. but there in case it gets troublesome again

File: src/styles/_colors.scss (hunk 3 above)
Comment (jacobrienstra) [reply src/styles/_colors.scss]: yeah, the other solution would be sort of, pushing everything one shade darker

**Pull request merged**